src/exporters/cds.py

```diff
@@ -280,23 +286,31 @@ def export(self,
             dataset = self.get_dataset(variable, granularity)
 
         if break_up:
-            p = multiprocessing.Pool(int(N_parallel_requests))
+            if parallel:  # Run in parallel
+                if N_parallel_requests < 1: N_parallel_requests = 1
+                p = multiprocessing.Pool(int(N_parallel_requests))
+
             output_paths = []
             for year, month in itertools.product(processed_selection_request['year'],
                                                  processed_selection_request['month']):
                 updated_request = processed_selection_request.copy()
                 updated_request['year'] = [year]
                 updated_request['month'] = [month]
 
-                # multiprocessing of the paths
                 output_paths.append(
-                    p.apply_async(
-                        self._export,
-                        args=(dataset, updated_request, show_api_request))
+                    self._export(dataset, updated_request, show_api_request)
                 )
+
+                if parallel:  # Run in parallel
```

Comment:
there are so many of these `if parallel` statements - might be better to have a separate function for calling it in parallel (just for ease of reading the code). What do you think?

Reply:
Hmm given that the functions would do the exact same thing (and require the same input arguments), just one in parallel and one not, I would prefer to keep it all together, rather than expect the user to remember two different function names.